### PR TITLE
Add test for converter and checks for null

### LINKED ISSUE #213: Improve test cases for convertor class
Hello!
Please improve test cases for convertor class.
Thats all. Many thanks.

### PR DESCRIPTION
**Pull request by camelya58**
Closes #213

### CODE REVIEW

File: src/main/java/com/reckue/post/utils/converters/NodeConverter.java [hunk 1]
```diff
@@ -18,7 +18,7 @@
      * @return the object of class Node
      */
     public static Node convert(NodeRequest nodeRequest) {
-
+        if (nodeRequest == null) throw new IllegalArgumentException("Null parameters are not allowed");
```
Comment (egnaf) [inline src/main/java/com/reckue/post/utils/converters/NodeConverter.java:21]: it's better this way:
```java
if (nodeRequest == null) {
    throw new IllegalArgumentException("Null parameters are not allowed");
}

```

File: src/test/java/com/reckue/post/utils/converters/ConverterTest.java [hunk 2]
```diff
@@ -5,18 +5,75 @@
...
+ *
+ * @author Kamila Meshcheryakova
+ */
 class ConverterTest extends PostServiceApplicationTests {
 
     @Test
-    void convert() {
-        Tag expected = Tag.builder()
+    void convertTagToTagResponse() {
+        Tag tag = Tag.builder()
```
Comment (egnaf) [inline src/test/java/com/reckue/post/utils/converters/ConverterTest.java:24]: Please in test use your custom classes.
For example, if you change tag class that you need change this class too. It's bad practice.

File: src/main/java/com/reckue/post/utils/Check.java [hunk 3]
```diff
@@ -0,0 +1,21 @@
+package com.reckue.post.utils;
```
Comment (egnaf) [inline src/main/java/com/reckue/post/utils/Check.java:1]: Move new classes from`src/main` to `src/test`.

Comment (camelya58) [reply src/main/java/com/reckue/post/utils/Check.java:1]: I can't do it because the annotations of Lombok doesn't work at these folders.  I've already tried.

Comment (egnaf) [reply src/main/java/com/reckue/post/utils/Check.java:1]: Then use without a lombok. Because this class will not be transferred to production after testing.
It is needed only in testing.

Comment (Hardelele) [reply src/main/java/com/reckue/post/utils/Check.java:1]: testCompileOnly 'org.projectlombok:lombok:1.18.12'
testAnnotationProcessor 'org.projectlombok:lombok:1.18.12'

Comment (egnaf) [reply src/main/java/com/reckue/post/utils/Check.java:1]: @camelya58 please try change scope of the lombok dependency.

Comment (egnaf) [reply src/main/java/com/reckue/post/utils/Check.java]: More info: https://projectlombok.org/setup/gradle

```gradle
compileOnly 'org.projectlombok:lombok:1.18.12'
annotationProcessor 'org.projectlombok:lombok:1.18.12'

testCompileOnly 'org.projectlombok:lombok:1.18.12'
testAnnotationProcessor 'org.projectlombok:lombok:1.18.12'
```
